config/data.js

```diff
@@ -47,22 +45,49 @@ module.exports = async (deps) => {
     const currencies = ['USD', 'EUR'];
     const marketTypes = [0, 1];
 
-    await Promise.all(addresses.slice(1, 5).map(async (address) => {
+    const offerReceipts = await Promise.all(addresses.slice(1, 5).map(async (address) => {
       const addOffer = deps.contracts.MetadataStore.methods.addOffer(
-        tokens[Math.floor(Math.random()*tokens.length)],
+        tokens[1],
+        // TODO un hardcode token and check below
+        // tokens[Math.floor(Math.random() * tokens.length)],
         address,
-        locations[Math.floor(Math.random()*locations.length)],
-        currencies[Math.floor(Math.random()*currencies.length)],
-        usernames[Math.floor(Math.random()*usernames.length)],
-        [paymentMethods[Math.floor(Math.random()*paymentMethods.length)]],
-        marketTypes[Math.floor(Math.random()*marketTypes.length)],
+        locations[Math.floor(Math.random() * locations.length)],
+        currencies[Math.floor(Math.random() * currencies.length)],
+        usernames[Math.floor(Math.random() * usernames.length)],
+        [paymentMethods[Math.floor(Math.random() * paymentMethods.length)]],
+        marketTypes[Math.floor(Math.random() * marketTypes.length)],
         Math.floor(Math.random() * 100)
       );
 
       const gas = await addOffer.estimateGas({from: address});
       return addOffer.send({from: address, gas});
     }));
 
+    console.log('Creating escrows and rating them...');
+    const expirationTime = parseInt((new Date()).getTime() / 1000, 10) + 10000;
+    const FIAT = 0;
+    const val = 1000;
+
+    const buyerAddress = addresses[1];
+    await Promise.all(addresses.slice(2, 5).map(async (creatorAddress, idx) => {
+      const ethOfferId = offerReceipts[idx - 1 + 2].events.OfferAdded.returnValues.offerId;
```

Comment:
To know offer id you just have to use the offerReceipts length -1, that would give you all offer ids